homeassistant/components/nut/sensor.py

```diff
@@ -928,13 +928,23 @@
         device_class=SensorDeviceClass.HUMIDITY,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    "ambient.humidity.status": SensorEntityDescription(
+        key="ambient.humidity.status",
+        translation_key="ambient_humidity_status",
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
     "ambient.temperature": SensorEntityDescription(
         key="ambient.temperature",
         translation_key="ambient_temperature",
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         device_class=SensorDeviceClass.TEMPERATURE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    "ambient.temperature.status": SensorEntityDescription(
+        key="ambient.temperature.status",
+        translation_key="ambient_temperature_status",
+        entity_category=EntityCategory.DIAGNOSTIC,
+    ),
```

Comment:
I don't think there is an existing enum that matches. However I think you can use `SensorDeviceClass.ENUM` here and use `options=` to list them.

That way the automation editor should see the options and the user will be able to select from a drop-down them e.g. to trigger automations.

This is better than just having raw text where the user has no idea of possible future values.

Reply:
Reviewed NUT source code to confirm this is an exhaustive list of currently supported return values.  Updated NUT component sensor class to use SensorDeviceClass of ENUM, set options, and wrote lambda function.